### PR TITLE
username changes + lmod

### PR DESCRIPTION
**Pull request by wpoely86**
Changes:
- Use `username` everywhere for a username. Added dedicated grok pattern `USERNAME_PAT`.
- Add Lmod logging pattern

### CODE REVIEW

File: files/grok-patterns [hunk 1]
```diff
@@ -1,5 +1,6 @@
 USERNAME [a-zA-Z0-9_-]+
 USER %{USERNAME}
+USERNAME_PAT %{USERNAME:username}
```
Comment (stdweird) [inline files/grok-patterns]: don not add anything here. this is a file downloaded from somewhere

File: tests/logstash_2.0.0.conf [hunk 2]
```diff
@@ -16,7 +16,7 @@ filter {
 
         match => {
             # RSYSLOGCUSTOM always last (and no PREFIX)!
-            "message" => ["%{RSYSLOGPREFIX}%{SSH_MSG}", "%{RSYSLOGPREFIX}%{MODULECMD_MSG}", "%{RSYSLOGPREFIX}%{NFS_MSG}", "%{RSYSLOGPREFIX}%{CEPH_MSG}", "%{RSYSLOGPREFIX}%{OPENNEBULA_MSG}", "%{RSYSLOGPREFIX}%{JUBE_MSG}", "%{RSYSLOGPREFIX}%{KEYVALUE_MSG}", "%{RSYSLOGPREFIX}%{QUATTOR_MSG}", "%{RSYSLOGPREFIX}%{SNOOPY_MSG}", "%{RSYSLOGCUSTOM}"]
+            "message" => ["%{RSYSLOGPREFIX}%{SSH_MSG}", "%{RSYSLOGPREFIX}%{MODULECMD_MSG}", "%{RSYSLOGPREFIX}%{LMOD_MSG}", "%{RSYSLOGPREFIX}%{NFS_MSG}", "%{RSYSLOGPREFIX}%{CEPH_MSG}", "%{RSYSLOGPREFIX}%{OPENNEBULA_MSG}", "%{RSYSLOGPREFIX}%{JUBE_MSG}", "%{RSYSLOGPREFIX}%{KEYVALUE_MSG}", "%{RSYSLOGPREFIX}%{QUATTOR_MSG}", "%{RSYSLOGPREFIX}%{SNOOPY_MSG}", "%{RSYSLOGCUSTOM}"]
```
Comment (stdweird) [inline tests/logstash_2.0.0.conf]: this needs a change in quattor when we will deploy it

File: files/grok-patterns (hunk 1 above)
Comment (boegel) [inline files/grok-patterns]: shouldn't it be mentioned in the file then as a comment?

**Comment by wpoely86** [comment]
@stdweird done. Tests pass on my laptop.

**Pull request merged**